src/bundle/Resources/public/scss/_modals.scss

```diff
@@ -79,3 +79,20 @@
     }
 }
 
+.ez-modal--select-modified-range,
```

Comment:
I think, the code in this file should be a bit reorganized. To keep the structure.